### DISCUSSION TITLE
Welcome to spring-boot-starter-calma Discussions!

### DISCUSSION THREAD

**Discussion by marvinSpring** [marvinSpring/spring-boot-starter-calma#9] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by grainGods** [comment]
本项目其实实用性可以，但是健壮性和扩展性一般
